CHANGES.md

```diff
@@ -1,5 +1,21 @@
 Change Log
 ==========
+### 1.38 - 2017-10-01
+
+* Breaking changes
+
+* Deprecated
+  * `HeadingPitchRoll.fromDirectQuaternion` is deprecated and his behaviour will replace `HeadingPitchRoll.fromQuaternion` behaviour in 1.43. [#5666](https://github.com/AnalyticalGraphicsInc/cesium/issues/5666)
```

Comment:
Do we want to deprecate this 6 months out?  We normally do 1-3 releases.  I think 3 releases would be appropriate for this case unless someone said otherwise.

Reply:
Well, who decides?

Reply:
@kaktus40 usually the developer and whoever reviews decides.

I agree with @hpinkos that 3 releases would be fine.  Please submit a GitHub issue for the deprecation.

For more on deprecation, see https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#deprecation-and-breaking-changes